File: Fontend/src/components/models/comment.ts
export interface Comment {
  id: number;
  text: string;
  star: number;
  bookingId: number;
  commentImages: [
    {
      id: number;
      image: string;
      commentId: number;
    }
  ];
}

export interface CommentPackage {
  id: number;
  text: string;
  star: number;
  bookingPackageId: number;
  commentPackageImages: [
    {
      id: number;
      image: string;
      commentPackageId: number;
    }
  ];
}

File: API/API/Models/Comment.cs
using System.Text.Json.Serialization;

namespace API.Models
{
    public class Comment
    {
        public int Id { get; set; }
        public string Text { get; set; }
        public double Star { get; set; }
        public int BookingId { get; set; }
        [JsonIgnore]
        public Booking Booking { get; set; }
        public ICollection<CommentImages> CommentImages { get; set; }
    }
}
